Backend/src/main/java/com/SYSC4806_Group13/SYSC4806_Project/Controllers/CartItemRestController.java

```diff
@@ -148,4 +149,20 @@ public Map<String, Integer> deleteUserCartItem(@RequestBody Map<String, Long> pa
         map.put("inventory", inventory);
         return map;
     }
+
```

Comment:
The name `updateListingInventory` as far as I can tell gives the wrong impression for what this method does. This listing inventory was already updated in the checkout method. Perhaps something like `updateAffectedCartItems` or `updateAssociatedCartItems` would make more sense

Reply:
Renamed to be more descriptive of how the listing inventory is affecting the cart quantity